### ISSUE TITLE
NeoForge mixin issue? Crash when pinging server from multiplayer menu

### ISSUE THREAD

**Issue by stascatto** [Aizistral-Studios/No-Chat-Reports#447]
### Modloader

NeoForge

### Minecraft Version

1.20.1

### Modloader Version

NeoForge 47.1.81

### No Chat Reports Version

1.20.1-v2.2.2

### The latest.log File

https://mclo.gs/XNYQd3J

### Bug Description

The game loads and plays perfectly fine but can't ping a server from the multiplayer menu, if left "Pinging" for long enough simply crashes. I don't know if I should report this on [NeoForge](https://github.com/neoforged/NeoForge/issues) instead but this is the only mod I've had such an issue with so far

### Steps to Reproduce

1. Install latest NeoForge and No Chat Reports for 1.20.1
2. Add a server in the Multiplayer menu
3. Wait for the crash

**Comment by Aizistral** [comment]
It sure is weird, but there's not much I can do about it until we switch from Forge to NeoForge. NeoForge stated that they will not attempt to maintain backwards compatibility with the original Forge, so it is likely that most mods designed to work with only one modloader will fail to function on another, at least from 1.20.2 and onward.

**Comment by P3rf3ctXZer0** [comment]
> It sure is weird, but there's not much I can do about it until we switch from Forge to NeoForge. NeoForge stated that they will not attempt to maintain backwards compatibility with the original Forge, so it is likely that most mods designed to work with only one modloader will fail to function on another, at least from 1.20.2 and onward.

That is correct. They said they will also being slowing moving from forge to some other name patch by patch. (Nothing that should break NeoMods just for reasons. I am sure its to stop getting users confused. I suggested Fox Framework ;) )

**Comment by MacChuck** [comment]
Same issue. Neoforge is supposed to be compatible at 1.20.1 with forge, but I'm assuming this is an edge case.

**Comment by P3rf3ctXZer0** [comment]
~~This is no longer a bug on latest Neoforge 47.1.96~~

**Comment by MacChuck** [comment]
Ohh good to know! Probably will fix my issue here? (I think it's the same problem, I found this report after I posted a new issue a few weeks ago) https://github.com/Aizistral-Studios/No-Chat-Reports/issues/460

**Comment by P3rf3ctXZer0** [comment]
@Aizistral it seems like it could be a long while also they keep pushing updates to neoforge 1.20.1 in meantime. Since most modders want to wait till whatever version of Neoforge 1.20.X falls on for final update I also heard minecraft vanilla might skip 1.20.5 now all together because the update size is closer to a content pack so it may be we get 1.21.0 either right away or a few short weeks after 1.20.5 that being said you should give LTS support for neoforge 1.20.1 until it stabilizes on a different version.

https://mclo.gs/H1glY0B

It says forge but I am on neoforge.

**Comment by P3rf3ctXZer0** [comment]
I found a simple work around if dev is waiting or if they can't fix it. Use Packmenu and set it up so clicking multiplayer goes straight to the server. Or just be faster than 35 secs on screen.

**Comment by P3rf3ctXZer0** [comment]
https://github.com/Aizistral-Studios/No-Chat-Reports/issues/460

**Comment by Seba244c** [comment]
Same issue https://mclo.gs/EVxT6Ce
